From c7c50134f2312281423cf4a4db4fd118f39ab464 Mon Sep 17 00:00:00 2001
From: Maarten Maathuis <madman2003@gmail.com>
Date: Mon, 18 Aug 2008 16:28:33 +0200
Subject: [PATCH 20/36] exa: remove useless cache{X,Y}off from UploadToScreen() arguments.
 (cherry picked from commit bc4cef1d0ae7cbb8f5fe16d576db45abd4330370)

---
 exa/exa_glyphs.c |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/exa/exa_glyphs.c b/exa/exa_glyphs.c
index ff665d5..59ef394 100644
--- a/exa/exa_glyphs.c
+++ b/exa/exa_glyphs.c
@@ -389,9 +389,10 @@ exaGlyphCacheUploadGlyph(ScreenPtr         pScreen,
     if (!pCachePixmap)
 	return FALSE;
 
+    /* CACHE_{X,Y} are in pixmap coordinates, no need for cache{X,Y}off */
     if (!pExaScr->info->UploadToScreen(pCachePixmap,
-				       CACHE_X(pos) + cacheXoff,
-				       CACHE_Y(pos) + cacheYoff,
+				       CACHE_X(pos),
+				       CACHE_Y(pos),
 				       pGlyph->info.width,
 				       pGlyph->info.height,
 				       (char *)pExaPixmap->sys_ptr,
-- 
1.6.0.3

